getting_started/hello_world_bash.Rmd

```diff
@@ -35,38 +52,12 @@ URL=http://www.data-intuitive.com/viash_docs/examples/hello_world/config.vsh.yam
 viash run $URL -- NAME. --greeter="Hello there,"
 ```
 
-## How does the Hello World component work?
+## How Does the Hello World Component Work?
```

Comment:
Section titles should be capitalised like sentences ("This is a short sentence."). Only the titles of documents should be maximally capitalised ("This is a Short Sentence"). Can you change this in the whole repo?

Reply:
Nice catch! Had het gisteren ook opgemerkt dat dit bij sommige documenten verkeerd zat.
Dit is nu in orde. Ik doe zometeen een commit.